modules/account-lib/src/coin/cspr/ifaces.ts

```diff
@@ -44,8 +44,7 @@ export interface Owner {
   weight: number;
 }
 
-export interface RunTimeArg {
-  action: CLValue;
-  weight: CLValue;
-  account?: CLValue;
-}
+export type ContractArgs = Record<
+  'action' | 'deployment_threshold' | 'key_management_threshold' | 'accounts' | 'weights',
```

Comment:
@CapnMigraine
I just realized that even if this was a typo before (thereshold instead of threshold),
this is how it's defined on the Wallet Initialization Contract we got from CasperLabs.

Therefore the Wallet Initialization is failing right now due to this change.
How should we proceed on this?

Reply:
OK, Let's keep the typo and add a comment for it